apps/public-docsite-v9/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@fluentui/upgrade-examples-v8-v9",
+  "name": "@fluentui/public-docsite-v9",
```

Comment:
nit: Can we rename this to `@fluentui/public-docsite-react-components` and the folder name to `public-docsite-react-components` to stay consistent with our other apps' naming conventions (such as vr-tests, ts-minbar etc.)?

Reply:
very true, i'm not being consistent. i guess the question is if v9 SHOULD be used in those other cases or not

Reply:
v9 is more descriptive than 'react-components', which is a placeholder package for v9 because we want to keep v8 as 'react'. 

I know the plan is to eventually move back to the 'react' name, maybe in v10. So maybe v9 is less confusing for all of the apps targeting the newer version?

Reply:
Yeah initially I wasn't fond of following the v9 convention for naming because I didn't want other packages names to be dependent on a version number that could change but now knowing that v9 is meant to be moved back to the `react` name, then i'm fine with it. Plus it's the more commonly used name when referring to this library by our team and partner teams anyway 😄

Reply:
My only concern is that future outside users/contributors might necessarily know the lingo that v9 is referring to `react-components`

Reply:
i think we have enough agreement that v9 is correct for now. We can easily revisit this later